decision-log/readme.md

```diff
@@ -0,0 +1,79 @@
+# Design decision log
```

Comment:
This would also fold well into our own internal engagement process. These ADR's could be real/living artifacts for checkpoints. Wondering if theres also room for content in our internal wiki for how this could help with some of the internal CSE rituals? Thoughts? Maybe that would give you a place to describe how this could be used during ADS, Game Plan, and Checkpoints.

Reply:
We also presented the decision log in the Checkpoint meetings in one of our engagements and it worked well. It would be great to see this as a practice across CSE.